### PR TITLE
Feature/ccclxxxvi 2

### PR DESCRIPTION
**Pull request by tavor118**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=993)

## Code reviewers

- [ ] @Andriy97091
- [ ] @irynazaiets
- [ ] @jelysaw
- [ ] @MaksymGromyk
- [ ] @shado1997
- [ ] @tavor118
- [ ] @VolodymyrBodnar
- [ ] @wm139

### Second Level Review

- [ ] @lhalam
- [ ] @alutsysh

## Summary of issue

Add edit and delete functionality for current.

## Summary of change

Added current model.
Added edit and delete functionality for current.
Added views and templates for current.
Added model and css for icons.
Added helper class for interacting with a database via pool manager. 

## Testing approach 

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: moneta/src/python/core/db/db_helper.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# pylint: disable=too-few-public-methods
+""" Module for requests using a pool manager. """
+
+import core.db.pool_manager as db
+from core.decorators import retry_request
+
+
+class DbHelper:
```
Comment (alutsysh) [inline moneta/src/python/core/db/db_helper.py]: class DbHelper():

File: moneta/src/python/core/db/db_helper.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    """
+    Class for interacting with database using a pool manager.
+    """
+    @staticmethod
```
Comment (alutsysh) [inline moneta/src/python/core/db/db_helper.py:15]: Why staticmethod?

File: moneta/src/python/db/current.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+    # TODO Vasyl # pylint: disable=fixme
+    @staticmethod
+    def create_current():
+        """
+
+        :return:
+        """
+        sql = f"""
+            START TRANSACTION;
```
Comment (alutsysh) [inline moneta/src/python/db/current.py]: What???? Transaction should be managed in db pool manager

File: moneta/src/python/db/current.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                 image_id - image for current
+        :return: True if success, else False
+        """
+        sql = f"""
+            UPDATE current  
+            SET name='{name}', mod_time={mod_time}, image_id={image_id}
+            WHERE current.id={current_id}; 
+            """
+        try:
+            Current._make_transaction(sql)
```
Comment (alutsysh) [inline moneta/src/python/db/current.py]: where args?

File: moneta/src/python/db/current.py [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+        Deletes connection between a user and a current in a database.
+        :params: user_id - id of logged user, current_id - id of  current,
+        :return: True if success, else False
+        """
+        sql = f"""
+            DELETE FROM user_current 
+            WHERE current_id={current_id} AND user_id={user_id};
+            """
+        try:
+            Current._make_transaction(sql)
```
Comment (alutsysh) [inline moneta/src/python/db/current.py]: args!

File: moneta/src/python/db/current.py [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+        query = query[0]
+        current = {
+            'current_id': query['id'],
+            'name': query['name'],
+            'currency': query['currency'],
+            'mod_time': query['mod_time'],
+            'amount': query['amount'],
+            'image_css': query['css'],
+            'can_edit': query['can_edit']
+        }
```
Comment (alutsysh) [inline moneta/src/python/db/current.py]: Why? you already have a dict

File: moneta/src/python/db/storage_icon.py [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+            WHERE category='{category}';
+            """
+        query = StorageIcon._make_select(sql)
+        icon_list = []
+        for row in query:
+            icon_list.append({
+                'icon_id': row['id'],
+                'css': row['css'],
+                'category': row['category']
+            })
```
Comment (alutsysh) [inline moneta/src/python/db/storage_icon.py]: What reason?

File: moneta/www/views/current.py [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+# TODO # pylint: disable=fixme
+# default user id while we don't have login/logout
+# Delete after adding login/logout
+USER_ID_FOR_DEBUG = 1
+
+
+def current_list(request):
+    """View for a current list."""
+    cur_list = Current.get_current_list_by_user_id(USER_ID_FOR_DEBUG)
```
Comment (alutsysh) [inline moneta/www/views/current.py]: what USER_ID_FOR_DEBUG? You have a real user!

**Review by alutsysh — APPROVED** [review]

**Review by VolodymyrBodnar — APPROVED** [review]

**Review by jelysaw — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**